driver-reactive-streams/src/main/com/mongodb/reactivestreams/client/internal/OperationExecutorImpl.java

```diff
@@ -63,40 +68,32 @@
         notNull("readPreference", readPreference);
         notNull("callback", callback);
         final SingleResultCallback<T> errHandlingCallback = errorHandlingCallback(callback, LOGGER);
-        clientSessionHelper.withClientSession(session, this, new SingleResultCallback<AsyncClientSession>(){
-            @Override
-            public void onResult(final AsyncClientSession clientSession, final Throwable t) {
-                if (t != null) {
-                    errHandlingCallback.onResult(null, t);
-                } else {
-                    getReadWriteBinding(readPreference, readConcern, clientSession,
-                            session == null && clientSession != null,
-                            new SingleResultCallback<AsyncReadWriteBinding>() {
```

Comment:
I can't tell if anything actually changed here.  Did this just get lambda-fied?

Reply:
That was just lambdafied.

Reply:
just lambda-fied